addons/mail/static/src/models/chatter/chatter.js

```diff
@@ -254,6 +239,9 @@ registerModel({
          * @private
          */
         _prepareAttachmentsLoading() {
+            if (this.isDisabled) {
+                return;
+            }
```

Comment:
Should be guarded sooner in the flow

Reply:
This is onchange method, there is no "sooner" possible.

Reply:
I guess I could just move it one level up, in the onchange itself, if that is what you meant?

Reply:
Yes, that's why I meant, just one level higher.
Was just strange to call `_prepareAttachmentsLoading` and just ignore it.

Reply:
I think the check is useless actually, `thread.isLoadingAttachments` should necessarily be false if disabled, so we should always fall on the first condition of the onchange.
I'm not sure why I added it, but probably a remaining of intermediary dev.